sys-power/tlp/tlp-1.9.0.ebuild

```
@@ -0,0 +1,83 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..14} )
+
+inherit udev optfeature python-single-r1
+
+DESCRIPTION="Optimize laptop battery life"
+HOMEPAGE="https://linrunner.de/tlp/"
+SRC_URI="https://github.com/linrunner/TLP/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/TLP-${PV}"
+
+LICENSE="GPL-2"
+SLOT="1"
+KEYWORDS="amd64"
+REQUIRED_USE="pd? ( ${PYTHON_REQUIRED_USE} )"
+
+IUSE="
+	+pd
+	+rdw
+"
+
+RDEPEND="
+	dev-lang/perl
+	virtual/udev
+	sys-apps/hdparm
+	net-wireless/iw
+	!app-laptop/laptop-mode-tools
+	sys-apps/pciutils
+	!sys-apps/tuned
+	sys-apps/usbutils
+	pd? ( dev-python/dbus-python )
+	pd? ( !sys-power/power-profiles-daemon )
+	pd? ( dev-python/pygobject )
+	pd? ( ${PYTHON_DEPS} )
+	pd? ( !sys-apps/tuned )
```

You could have this inside single parentheses, 
```suggestion
    pd? (
        !sys-apps/tuned
        !sys-power/power-profiles-daemon
        ${PYTHON_DEPS}
        dev-python/dbus-python
        dev-python/pygobject
    )
```

Or
```suggestion
    pd? ( !sys-apps/tuned
        !sys-power/power-profiles-daemon
        ${PYTHON_DEPS}
        dev-python/dbus-python
        dev-python/pygobject )
```

Note that the **dev-python** packages are missing their **[${PYTHON_USEDEP}]** dependencies.